aws/resource_aws_ecr_lifecycle_policy_test.go

```diff
@@ -0,0 +1,97 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ecr"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSEcrLifecyclePolicy(t *testing.T) {
```

Comment:
Nitpick, but we like to give all basic test functions a suffix `_basic` so that when we add more in the future we can still run just this function without matching other ones which may have different suffix.